Library/Formula/llvm.rb

```diff
@@ -118,6 +118,8 @@ def install
       args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
     end
 
+    args << "-DCMAKE_CXX_FLAGS=#{ENV['CPPFLAGS']}" unless OS.mac?
```

Comment:
Please report the exact error that it fixes in a Ruby comment. It helps googling and maintenance.